robottelo/api/utils.py

```diff
@@ -62,6 +62,8 @@ def enable_rhrepo_and_fetchid(basearch, org_id, product, repo,
         payload['basearch'] = basearch
     if releasever is not None:
         payload['releasever'] = releasever
+    if product is not None:
```

Comment:
do not think that product can be None here as we have (and if the product does not exit it should fail here):
```python
 product = entities.Product(name=product, organization=org_id).search()[0]
```
perhaps should be sufficient only:
```python
payload['product_id'] = product.id
```